backend/app.js

```diff
@@ -15,7 +60,7 @@ app.use(
   })
```

Comment:
all app.use lines should be at top of file. I think `app.use(cors())` is in this file twice